api/src/client/json-profiles/license-status/response.json

```diff
@@ -2,7 +2,7 @@
   {
     "fullName": "",
     "addressLine1": "",
-    "addressCity": "",
+    "foreignAddressCity": "",
```

Comment:
this isn't correct, shouldn't be changed. this is a different field / functional altogether. This is in license-search and you are only changing formation